mlrun/projects/project.py

```diff
@@ -1964,6 +1965,7 @@ def run(
                           https://apscheduler.readthedocs.io/en/3.x/modules/triggers/cron.html#module-apscheduler.triggers.cron
                           for using the pre-defined workflow's schedule, set `schedule=True`
         :param timeout:   timeout in seconds to wait for pipeline completion (used when watch=True)
+        :param overwrite_schedule: delete schedule when submitting a new one with the same name
```

Comment:
I don't think the user minds about the underhood implementation (That we are currently deleting, but in the future this will be implemented with `update` so I would just mention what is going to be done. you can use words like `replace` etc...